pkg/kubelet/cm/cpumanager/topology/topology_test.go

```diff
@@ -1100,3 +1144,39 @@ func TestCPUNUMANodeID(t *testing.T) {
 		})
 	}
 }
+
+func TestCPUsPerSocket(t *testing.T) {
```

Comment:
this will conflict with https://github.com/kubernetes/kubernetes/pull/124561 - and 124561 is more correct IMO

Reply:
resolved in 512ca4b9888c303b1b0aef5469293a099bc4cd9b